SimplenoteUITests/NoteList.swift

```diff
@@ -212,14 +231,27 @@ class NoteListAssert {
         NoteListAssert.noteListShown(forSelection: UID.NavBar.trash)
     }
 
+    class func noteContentIsShown(_ note: NoteData) {
```

Comment:
Nitpick. Again, thinking about readability, I wonder if this works best

```suggestion
    class func contentIsShown(for note: NoteData) {
```

Which would be used like `NoteListAssert.contentIsShown(for: godzilla)`.

"note list assert content is shown for godzilla" may read a bit better than "note list assert not content is shown godzilla".

When I say "I wonder if..." I really am wondering. The answer can be no, I'm fine with that.

Reply:
Applied your change. As you said, this is a matter of preference, so I'd like to try how it feels